src/include/uapi/drm_local/amdxdna_accel.h

```diff
@@ -62,6 +62,7 @@ extern "C" {
 #define	AMDXDNA_DEV_TYPE_UNKNOWN	-1
 #define	AMDXDNA_DEV_TYPE_KMQ		0
 #define	AMDXDNA_DEV_TYPE_UMQ		1
+#define	AMDXDNA_DEV_TYPE_VE		2
```

Comment:
The AMDXDNA_DEV_TYPE_* is used in shim. It is a flag that control the behavior of the SHIM.

I suggest we add this line in the first PR for SHIM code of ve2 if needed.

AMDXDNA_DEV_TYPE_KMQ means the device support kernel managed queue. SHIM should send commands to kernel to execute AIE task.
AMDXDNA_DEV_TYPE_UMQ means the device support user managed queue. SHIM do NOT send commands via kernel.

If above two definition doesn't fit the VE2 usecase, please let me know.